docker-compose/jaeger/docker-compose.yaml

```diff
@@ -4,35 +4,41 @@ services:
   # (https://www.jaegertracing.io/docs/1.17/getting-started/#all-in-one) for distributed tracing.
   #
   # Disk: none
-  # Ports exposed to other Sourcegraph services: 5778/TCP 6831/UDP 6832/UDP 14250/TCP
+  # Ports exposed to other Sourcegraph services: 5778/TCP 6831/UDP 6832/UDP 14250/TCP 4320/UDP 4321/UDP
   # Ports exposed to the public internet: none
   # Ports exposed to site admins only: 16686/HTTP
   #
   jaeger:
     container_name: jaeger
-    image: 'index.docker.io/sourcegraph/jaeger-all-in-one:216430_2023-05-02_5.0-3cc9006de32c@sha256:ec73cff6ea398d96241a9451634fc83682292b0175cc63c09f1f866cf03beb8d'
+    image: 'index.docker.io/sourcegraph/jaeger-all-in-one:281057_2024-07-04_5.4-b98c7d084d30@sha256:3ff2c29a456f4b275a46ca413eb38c84ee9df6109460691b13a798aecee038a0'
     cpus: 0.5
     mem_limit: '512m'
     ports:
       # Query port
       - '0.0.0.0:16686:16686'
       # Collector port
       - '0.0.0.0:14250:14250'
+      - '0.0.0.0:4320:4320' # gRPC port
+      - '0.0.0.0:4321:4321' # HTTP port
```

Comment:
@bobheadxi Are these the right ports? The [announcement blog](https://medium.com/jaegertracing/introducing-native-support-for-opentelemetry-in-jaeger-eb661be8183c) mentions `4317` and `4318`, but the [PR in the monorepo](https://github.com/sourcegraph/sourcegraph/pull/63171/files#diff-bb8e3383b3d6ce6cc850d8aea2c04d5c800914ad30da787ccca0bdab8e0fc3b4R1271) has `4320` and `4321`

I've used these across the other PRs I made too.

Reply:
Locally the port must be different, otherwise it will conflict with a locally running OTEL-collector, which uses 4317 and 4318 by default as well